src/Notifier.cpp

```diff
@@ -52,8 +54,8 @@ bool Notifier::Initialize(const size_t handshakeCount) {
       gQueue.reset(new Shotgun);
       try {
          gQueue->Aim(GetNotifierQueueName());
-      } catch (std::exception& e) {
-         LOG(WARNING) << "Caught exception: " << e.what();
+      } catch (std::string& e) {
```

Comment:
You are completely right. I stand corrected.
https://github.com/weberr13/QueueNado/blob/master/src/Shotgun.cpp

In that case I would prefer if you corrected this bizarre Shotgun throw to instead use a std::exception